TheFourFifths/consus-client
# Client student upload tff106

### Summary
You can upload students.

### Checklist


### How to Review

1. Start server on parse-file-tff107
2. Start client
3. Click the upload students link on main page
4. Select file which is in docs/Mock_students.xlsx
5. Click submit
6. Verify toast appears for uploading students
7. Enter id of newly created student
8. Verify checkout page shows the new student

### Links

- [TFF-106](https://msoese.atlassian.net/browse/TFF-106)
- TheFourFifths/consus/pull/44
